addons/mail/models/res_config_settings.py

```diff
@@ -39,6 +39,19 @@ class ResConfigSettings(models.TransientModel):
     primary_color = fields.Char(related='company_id.primary_color', string="Header Color", readonly=False)
     secondary_color = fields.Char(related='company_id.secondary_color', string="Button Color", readonly=False)
 
+    tenor_api_key = fields.Char('Tenor API key', config_parameter='discuss.tenor_api_key')
+    tenor_content_filter = fields.Selection(
+        [('high', 'High'),
+        ('medium', 'Medium'),
+        ('low', 'Low'),
+        ('off', 'Off')],
+        string='Tenor content filter',
```

Comment:
Could be nice to explain what each do in `help=`, or refer to some tenor GIF documentation